brainiak/isc.py

```diff
@@ -1097,6 +1109,9 @@ def permutation_isc(iscs, group_assignment=None, pairwise=False,  # noqa: C901
     n_permutations : int, default: 1000
         Number of permutation iteration (randomizing group assignment)
 
+    side : str, default: 'two-sided'
+        Perform one-sided ('left' or 'right') or 'two-sided' test
```

Comment:
The `side` default described in this docstring is not the default assigned in the function declaration.

More generally, we should not describe defaults in the docstrings because their presence in declarations makes them visible both in the Python built-in `help` function and the HTML docs:
```
python3 -c "import brainiak.isc; help(brainiak.isc.permutation_isc)"
```
https://brainiak.org/docs/brainiak.html#brainiak.isc.permutation_isc